README.md

```diff
@@ -18,97 +18,39 @@
+
+Want to learn more about Atlas? Visit our [🌐 Website](https://atlasos.net/) and [📚 Documentation](https://docs.atlasos.net/).
+
+## 🤔 What is Atlas?
+Atlas is an open-source project that enhances Windows by eliminating factors that negatively impact gaming performance. It's the perfect choice for reducing system and network latency, and input lag, while maintaining system privacy and prioritizing performance.
+  
+## 👀 Why Atlas?
+### 🔒 Enhanced Privacy
+Atlas removes all types of tracking embedded within Windows and implements numerous group policies to minimize data collection. However, it cannot ensure the security for things outside the scope of Windows, such as browsers and other third-party applications.
+  
+### 🛡️ Increased Security
+Unlike other custom Windows ISOs, Atlas is safer as it doesn't require downloading a modified Windows ISO from the internet. Atlas also includes optional security features (e.g. Defender, Update) that other custom Windows OSs often strip.
+  
+### ✅ Optimized Performance
+Atlas strikes a balance between performance and compatibility. It implements numerous changes to improve Windows performance without breaking essential features.
+  
+### 🔒 Legal Compliance
+Atlas partners with the Ameliorated team to provide users with a safer and legal way to install, using [AME Wizard](https://ameliorated.io). With it, Atlas fully complies with [Microsoft's Terms of Service](https://www.microsoft.com/en-us/Useterms/Retail/Windows/10/UseTerms_Retail_Windows_10_English.htm).
+
+### 🌐 Open Source
+All scripts used by Atlas are open-sourced in this repository. Executables included in the playbook are open-sourced in our [utilities](https://github.com/Atlas-OS/utilities) repository. 
+
+The packaged Atlas playbook (`.apbx` - AME Wizard script package) can be viewed as a .7z archive, with the password `malte` (standard for AME Wizard playbooks). This is solely to bypass false flags from antiviruses.
+  
 ## 🎨 Brand kit
-Feeling creative? Want to create your own Atlas wallpaper with some original creative designs? Our brand kit has got you covered!
-Anyone can access the Atlas brand kit—you can download it [here](https://github.com/Atlas-OS/branding/archive/refs/heads/main.zip) and make something spectacular!
-
-We also have a dedicated area on our [forum](https://forum.atlasos.net/t/art-showcase), so you can share your creations with other creatives and maybe even spark some inspiration! You can also find creative wallpapers that other users share here too!
+Want to create your own Atlas wallpaper with some original creative designs? Download our brand kit [here](https://github.com/Atlas-OS/branding/archive/refs/heads/main.zip) and share your creations on our [forum](https://forum.atlasos.net/t/art-showcase).
+  
+## ⚠️ Disclaimer
+Atlas complies with the Windows EULA; activation in Windows is not modified. A valid legitimate license is required to install Atlas.
```

Comment:
Activation will no longer be a requirement for AME Wizard in v0.4.0. That was mostly used to ensure that the theme set correctly, but it isn't really a concern as I'm personally going to be reimplementing things.

Remove the parts about where to buy a key and that it's required. Instead, state that Atlas doesn't redistribute any Windows ISOs.